curriculum/templates/curriculum/lessonplan_detail.html

```diff
@@ -14,142 +14,132 @@
+                    <div class="btn-group mt-2" role="group" aria-label="Actions for lesson plan author">
+                        <a href="{% url 'update-lesson-plan' pk=lessonplan.id %}" class="px-2 py-0 btn btn-outline-secondary lessonplan-action-button">Edit</a>
+                        <a href="{% url 'delete-lesson-plan' pk=lessonplan.id %}" class="px-2 py-0 btn btn-outline-danger lessonplan-action-button">Delete</a>
                     </div>
+                    {% endif %}
                 </div>
-                <div class="list-group-item">
-                    <h1 class="lead">Materials</h1>
-                    <p>{{ lessonplan.materials }}</p>
-                    <h1 class="lead">Digital Files</h1>
-                    {% if lessonplan.resources.all or lessonplan.resource_links.all %}
-                        {% for resource in lessonplan.resources.all %}
-                            {% spaceless %}
-                                <p>
-                                    <a href="{% url 'detail-lesson-resource' pk=resource.id %}" download="{{ resource.file.name }}" target="_blank">{{ resource.name }}</a>
-                                    <span class="ml-2"> ({{ resource.get_semantic_type_display }})</span>
-                                </p>
-                            {% endspaceless %}
-                        {% endfor %}
-                        {% for resource in lessonplan.resource_links.all %}
-                            {% spaceless %}
-                                <p>
-                                    <a href="{{ resource.link }}" target="_blank">{{ resource.name }}</a>
-                                    <span class="ml-2"> ({{ resource.get_semantic_type_display }})</span>
-                                </p>
-                            {% endspaceless %}
-                        {% endfor %}
-                    {% else %}
-                        <span>There are no files associated with this lesson plan</span>
+                <p class="mb-4 stars text-primary" data-average-rating="{{ average_rating }}" >
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                    <i class="fa-star" aria-hidden="true"></i>
+                </p>
+                <div class="col-md-8 pl-0">
+                    <p>{{ lessonplan.summary }}</p>
+                </div>
+                <div>
+                    <span>
+                        <span class="sr-only">In-class timeline: </span>
+                        <span class="badge badge-outlined badge-secondary lessonplan-info">{% number_to_words lessonplan.num_classes True %} {% format_duration lessonplan.single_class_time True %} class{{lessonplan.num_classes|pluralize:'es'}}</span>
```

Comment:
Maybe we could add icons to make them easier to differentiate, what do you think?

Reply:
A question for @cyan478